samples/word/04-range/split-words-of-first-paragraph.yaml

```diff
@@ -0,0 +1,161 @@
+id: word-basics-split-words-of-first-paragraph-
+name: Range manipulation
+description: 'Show how to split a paragraph into word ranges and then traverses all the ranges to format each word, producing a "karaoke" effect.'
+author: Microsoft
+host: WORD
+api_set:
+    WordApi: 1.1
+script:
+    content: |-
+        $("#setup").click(setup);
+        $("#highlight").click(highlightWords);
+
+        function highlightWords() {
+            Word.run(function (context) {
+                // var myWords = context.document.body.paragraphs.first.split([" "], true, true);
+                var myPars = context.document.body.paragraphs;
+                context.load(myPars, { expand: 'font' });
```

Comment:
Why are you doing expand?  I have yet to see a place where it was actually needed, and a good idea.  Doing a regular load and only of the properties you need would be far better.